### ISSUE TITLE
Hybrid CPU+CUDA inference

### ISSUE THREAD

**Issue by drrros** [antirez/ds4#71]
Does ds4 have support (or plans?) for hybrid CPU\GPU (CUDA) inference?

**Comment by antirez** [comment]
I believe I'll not add any CPU-related inference if not the one that is already in that is used to compare the logits against a simple implementation. offloading layers to the CPU is going to be too slow anyway... this project wants to be very practical and narrow in scope, for all the rest we have llama.cpp that will be able to run the GGUFs on this repository soon I bet (you could already use my llama.cpp fork with the DeepSeek v4 Flash implementation).
